api/definitions.yaml

```diff
@@ -97,8 +97,31 @@ NewMessage:
       $ref: "#/MessageContent"
     default_addresses:
       $ref: "#/NewMessageDefaultAddresses"
+    payment_data:
+      $ref: "#/PaymentData"
   required:
     - content
+PaymentData:
+  type: object
+  description: |-
+    Metadata needed to process pagoPA payments.
+  properties:
+    amount:
+      $ref: "#/PaymentAmount"
+    notice_number:
+      $ref: "#/PaymentNoticeNumber"
+PaymentNoticeNumber:
+  description: The field ["Numero Avviso"](https://pagopa-specifichepagamenti.readthedocs.io/it/latest/_docs/Capitolo7.html#il-numero-avviso-e-larchivio-dei-pagamenti-in-attesa)
+    of pagoPa, needed to identify the payment. Format is `<aux digit (1n)>[<application code> (2n)]<codice IUV (15|17n)>`.
+    See [pagoPa specs](https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_pagamenti_1_3_1_0.pdf) for more info on this field and the IUV.
+  type: string
+  minLength: 16
```

Comment:
would it make sense to have a regex for validation?

Reply:
done (18 digits starting with either 0,1,2,3)